packages/components/src/components/popover/popover-styles.ts

```diff
@@ -75,40 +85,66 @@ const directionArrowMap: { [key in PopoverDirection]: JssStyle } = {
     left: 0,
     transform: 'translateY(-50%)',
     borderWidth: join(borderWidth, 0, borderWidth, borderWidth),
-    borderColor: join(transparentColor, transparentColor, transparentColor, backgroundColor),
+    borderColor: join(transparentColor, transparentColor, transparentColor, backgroundColorThemeLight),
     [mediaQueryForcedColors]: {
       borderColor: join(canvas, canvas, canvas, canvasText),
     },
   },
 };
 
-export const getComponentCss = (direction: PopoverDirection): string => {
-  const spacerBox = '-1rem';
+export const getComponentCss = (direction: PopoverDirection, theme: Theme): string => {
+  const spacerBox = '-16px';
+  const { hoverColor, focusColor } = getThemedColors(theme);
 
   return getCss({
     '@global': {
       ':host': {
         ...addImportantToEachRule({
           position: 'relative',
           display: 'inline-block',
+          // Needs to stay in rem for up-scaling
           width: '1.5rem', // width of icon (to improve ssr support)
```

Comment:
I think this value is not correct anymore, `fontLineHeight` util should be used instead, right?